### ISSUE TITLE
Unknown node: tnt:tnt

### ISSUE THREAD

**Issue by Xachman** [minetest/minetest#9667]
##### Minetest version

```
Minetest 5.2.0 (Linux)
BUILD_TYPE=Release
RUN_IN_PLACE=0
USE_GETTEXT=0
USE_SOUND=0
USE_CURL=1
USE_FREETYPE=0
USE_LUAJIT=0
STATIC_SHAREDIR="/usr/local/share/minetest"
```

##### OS / Hardware

Operating system:
Centos Docker
CPU:
Xeon e5620

GPU model:
OpenGL version:

##### Summary

```
ERROR[Main]: ServerError: AsyncErr: ServerThread::run Lua: Unknown node: tnt:tnt
```
I got this error when playing im not sure if I need a mod for this or if its supposed to be a standard node.
##### Steps to reproduce

I think it has to do with the TenPlus1 mobs_monsters mod.  Something shot a fireball at me and then I get this error.

**Comment by sfan5** [comment]
Very likely a duplicate of minetest/minetest_game#2634

You can either set `enable_tnt = true` (note that this will make TNT available to your players) or apply these [changes](https://github.com/minetest/minetest_game/pull/2636/files) to minetest_game.

**Comment by Xachman** [comment]
@sfan5 Thanks looks like thats it

**Comment by Xachman** [comment]
@sfan5 It looks like this may be fixed I switched to using the docker 5.2.0 do you think it would be better to in general stay on the master docker image?

**Comment by sfan5** [comment]
`master` is mostly stable and works just fine but for regular users I'd recommend staying on 5.2.0.
